### ISSUE TITLE
First load show white space

### ISSUE THREAD

**Issue by miztizm** [bumrufus/humhub-modules-quotes#2]
When first loading a quotes module its shows a white space 
when i click somewhere in humhub and comeback to front page, quotes work again.

**Comment by bumrufus** [comment]
yes thats a known issue with javascript, i will look into it again, maybe i will find a solution for that
